build.gradle

```diff
@@ -183,17 +187,17 @@ dependencies {
     providedCompile("org.eclipse.jetty.websocket:websocket-client:${jettyVersion}")
     providedCompile("org.eclipse.jetty.websocket:javax-websocket-server-impl:${jettyVersion}")
 
-    // The spring version was not upgraded to 4.3.30.RELEASE because of issues arising with openstack infrastructure
```

Comment:
Is this not risky for the openstack infrastructure and its users?

Reply:
I have tested the full patch running the cloud-tests on system-test platform yesterday. So normally, it should be ok